recipes/python-3.5/meta.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: python
+  version: 3.5.1
+
+source:
+  fn: Python-3.5.1.tgz
+  url: https://www.python.org/ftp/python/3.5.1/Python-3.5.1.tgz
+  md5: be78e48cdfc1a7ad90efff146dce6cfe
+  patches:
+    - osx-dist.patch       # [osx]
+    - win-find_exe.patch   # [win]
+
+build:
+  no_link: 
+    - bin/python
+  number: 0
+
+requirements:
+  build:
+    - bzip2     # [unix]
+    - zlib      # [unix]
```

Comment:
Zlib is downloaded by the Windows build script via SVN so there is no need to install it into the environment.